packages/web3-eth-iban/src/index.ts

```diff
@@ -15,218 +15,9 @@ You should have received a copy of the GNU Lesser General Public License
-			return toChecksumAddress(paddedBigInt);
-		}
-		throw new Error('Iban is indirect and cannot be converted. Must be length of 34 or 35');
-	};
-	/**
-	 * This method should be used to create an ethereum address from a direct iban address
-	 *
-	 * @param iban
-	 */
-	public static toAddress = (iban: string): HexString => {
-		const ibanObject = new Iban(iban);
-		return ibanObject.toAddress();
-	};
-
-	/**
-	 * Convert the passed BBAN to an IBAN for this country specification.
-	 * Please note that <i>"generation of the IBAN shall be the exclusive responsibility of the bank/branch servicing the account"</i>.
-	 * This method implements the preferred algorithm described in http://en.wikipedia.org/wiki/International_Bank_Account_Number#Generating_IBAN_check_digits
-	 *
-	 * @param bban
-	 */
-	public static fromBban(bban: string): Iban {
-		const countryCode = 'XE';
-
-		const remainder = this._mod9710(this._iso13616Prepare(`${countryCode}00${bban}`));
-		const checkDigit = `0${(98 - remainder).toString()}`.slice(-2);
-
-		return new Iban(`${countryCode}${checkDigit}${bban}`);
-	}
-
-	/**
-	 * This method should be used to create iban object from an ethereum address
-	 *
-	 * @param address
-	 */
-	public static fromAddress(address: HexString): Iban {
-		if (!isAddress(address)) {
-			throw new InvalidAddressError(address);
-		}
-
-		const num = BigInt(hexToNumber(address));
-		const base36 = num.toString(36);
-		const padded = leftPad(base36, 15);
-		return Iban.fromBban(padded.toUpperCase());
-	}
-
-	/**
-	 * This method should be used to create iban address from an ethereum address
-	 *
-	 * @param address
-	 */
-	public static toIban(address: HexString): string {
-		return Iban.fromAddress(address).toString();
-	}
-
-	/**
-	 * Should be used to create IBAN object for given institution and identifier
-	 *
-	 * @param options
-	 */
-	public static createIndirect(options: IbanOptions): Iban {
-		return Iban.fromBban(`ETH${options.institution}${options.identifier}`);
-	}
-
-	/**
-	 * Should be called to get client identifier within institution
-	 */
-	public client(): string {
-		return this.isIndirect() ? this._iban.slice(11) : '';
-	}
-
-	/**
-	 * Returns the ibans checksum
-	 */
-	public checksum(): string {
-		return this._iban.slice(2, 4);
-	}
-
-	/**
-	 * Returns institution identifier from iban
-	 */
-	public institution(): string {
-		return this.isIndirect() ? this._iban.slice(7, 11) : '';
-	}
-
-	/**
-	 * Should be called to check if iban is correct
-	 */
-	public isValid(): boolean {
-		return Iban._isValid(this._iban);
-	}
-
-	/**
-	 * This method should be used to check if given string is valid iban object
-	 *
-	 * @param iban
-	 */
-	public static isValid(iban: string) {
-		return Iban._isValid(iban);
-	}
-
-	public toString(): string {
-		return this._iban;
-	}
-
-	/**
-	 * check if iban number if indirect
-	 */
-	public isIndirect(): boolean {
-		return Iban.isIndirect(this._iban);
-	}
-}
+export default Web3Iban;
```

Comment:
```suggestion
export default Iban;
```
Make it compatible to 1.x